OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -266,7 +266,21 @@ string SysInfoQuery()
 				Uri.EscapeUriString(System.Globalization.CultureInfo.InstalledUICulture.TwoLetterISOLanguageName),
 				Uri.EscapeUriString(ModMetadata.AllMods["modchooser"].Version),
 				Uri.EscapeUriString(Game.ModData.Manifest.Mod.Id),
-				Uri.EscapeUriString(Game.ModData.Manifest.Mod.Version));
+				Uri.EscapeUriString(Game.ModData.Manifest.Mod.Version),
+				Uri.EscapeUriString(TestWritePermissions().ToString()));
+		}
+
+		byte TestWritePermissions()
+		{
+			try
+			{
+				using (var fs = File.Create(Path.Combine(".", "mods", ".testwritable"), 1, FileOptions.DeleteOnClose))
```

Comment:
To settle the argument you will also need to show that Windows isn't redirecting the write to the VirtualStore directory.